### PR TITLE
Fix typo in color files

### PR DESCRIPTION
**Pull request by HassanAlsamahi**
I've found some typos in hsv.py, in the commented summary section, so I fixed them, and added class example in the rest of color conversions files.

### CODE REVIEW

File: kornia/color/rgb.py [hunk 1]
```diff
@@ -17,6 +17,12 @@ class RgbToBgr(nn.Module):
         - image: :math:`(*, 3, H, W)`
         - output: :math:`(*, 3, H, W)`
 
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> bgr = kornia.image.RgbToBgr()
```
Comment (edgarriba) [inline kornia/color/rgb.py]: This is kornia.color

Comment (HassanAlsamahi) [reply kornia/color/rgb.py]: excuse me, I didn't understand the request. I just added the example form in every class in kornia.color

Comment (HassanAlsamahi) [reply kornia/color/rgb.py]: I think I got it, you mean instead of kornia.image.... it'll be kornia.color? sorry my bad I'll fix it

**Comment by dvd42** [comment]
@edgarriba LGTM

**Pull request merged**